src/contact.rs

```diff
@@ -796,7 +797,31 @@ impl Contact {
             Ok(row_id)
         }).await?;
 
-        Ok((ContactId::new(row_id), sth_modified))
+        let contact_id = ContactId::new(row_id);
+
+        Ok((contact_id, sth_modified))
+    }
+
+    /// Get all chats the contact is part of
+    pub async fn get_chats_with_contact(
```

Comment:
do we want to keep this method, even though we probably won't use it anymore in this pr: 
- it't to expensive to run in emitting events compared to telling the ui just to reload the whole list
- on the other hand this method could be useful on it's own maybe?